### DISCUSSION TITLE
What does the [favoriteThemes] entry do?

### DISCUSSION THREAD

**Discussion by Gravifer** [t1m0thyj/WinDynamicDesktop#343] [Q&A] [answered]
What's the significance of this entry in `settings.conf`?

**Comment by t1m0thyj** [comment, marked as answer]
Since WDD 4.5.0, you can right-click on a thumbnail in the Select Theme dialog and select "Add to favorites". This stars the theme and shows it at the top of the list. These favorited or starred themes are stored in the `favoriteThemes` config property.
![image](https://user-images.githubusercontent.com/22344007/122618489-5cf41200-d05c-11eb-9174-c6a11b71524c.png)

**Reply by Gravifer** [reply]
Thanks a lot! Didn't know about this ;)
